### DISCUSSION TITLE
Scene Viewer with webxr intent call Wall Placement

### DISCUSSION THREAD

**Discussion by memoricab** [google/model-viewer#2961] [Q&A] [answered]
Hello,

Sorry, this is not so related to the model-viewer but I'd appreciate it if you give me any hints.
I start Scene Viewer via Google Play Services android intent from my web page as below.
`intent://arvr.google.com/scene-viewer/1.0 file=location&mode=ar_preferred#Intent;scheme=https;package=com.google.android.googlequicksearchbox;action=android.intent.action.VIEW;S.browser_fallback_url=https://developers.google.com/ar;end;");`

The thing is I don't see any parameters in the intent link that it could place the model on the wall. Do you know how to pass surface placement into the intent link?

I also looked for related code maybe I could create a PR in googleAR but could not find it.

Thank you in advance

**Comment by memoricab** [comment, marked as answer]
Hmm I noticed that actually Scene Viewer places the model automatically on the wall. I don't know how it detects that model should be placed on the wall though, probably it is declared in model features to be placed on wall surface.

**Reply by ghost** [reply]
To answer the original question the parameter is `enable_vertical_placement=true`

**Reply by memoricab** [reply]
Thank you @timmmeh, could you please tell me where did you find this information?

**Reply by memoricab** [reply]
Ah probably here, https://github.com/google/model-viewer/blob/bc9899a9fdaf8e50e44354c746f52a31a8242a30/packages/model-viewer/src/features/ar.ts#L322

but when I'm checking Google documentation here https://developers.google.com/ar/develop/java/scene-viewer
There is no clue how to do it.

**Reply by ghost** [reply]
Correct, I did see it there, I also knew what to look for. 
Not sure why it's not documented there, I'm asking around to find out, you might to submit a bug against that page.

**Reply by memoricab** [reply]
Thanks @timmmeh , I will give feedback on that page.

**Reply by ghost** [reply]
PR submitted, documentation is updated, thanks for pointing this out.
